pkg/kubecli/kubecli.go

```diff
@@ -47,6 +47,10 @@ func Get() (*kubernetes.Clientset, error) {
 }
 
 func GetRESTClient() (*rest.RESTClient, error) {
+	return GetRESTClientFromFlags(master, kubeconfig)
```

Comment:
They are set after `flag.Parse()` was called.

That is a little bit ugly at the moment, because we have some code which does just kubecli.GetRESTClient() and to make that testable I am messing around with the `flag` package in the test ...